docs/config.md

```diff
@@ -41,7 +41,7 @@ for more details.
 | `AspNet` | ASP.NET Core | * | source |
 | `GraphQL` | [GraphQL](https://www.nuget.org/packages/GraphQL/) | ≥2.3.0 & < 3.0.0 | bytecode |
 | `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | * | source |
-| `MongoDb` | [MongoDB.Driver.Core](https://www.nuget.org/packages/MongoDB.Driver.Core) | ≥2.3.0 & < 3.0.0 | source & bytecode |
+| `MongoDB` | [MongoDB.Driver.Core](https://www.nuget.org/packages/MongoDB.Driver.Core) **Not supported on .NET Framework**  | ≥2.3.0 & < 3.0.0 | source & bytecode |
```

Comment:
```suggestion
| `MongoDb` | [MongoDB.Driver.Core](https://www.nuget.org/packages/MongoDB.Driver.Core) **Not supported on .NET Framework** | ≥2.3.0 & < 3.0.0 | source & bytecode |
```

here is why: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/3d6d2bbfc20a82c9331aa8a16457904cb85b7506/src/OpenTelemetry.AutoInstrumentation/Configuration/TracerInstrumentation.cs#L47

Reply:
I agree. Just do not forget to update the examples as well if you decide to do it here 😉

Reply:
Let's do it as a follow up - I can spare some minutes to do it today.